src/Week-calendar/CalendarCells.tsx

```diff
@@ -0,0 +1,11 @@
+import styles from './week-calendar.module.css';
+
+const CalendarCells = () => (
+  <div className={styles.calendarCells} id="calendarCells">
+    {new Array(7 * 19).fill(undefined).map((_, i) => {
```

Comment:
idk what 7 and 19 is 🤷‍♀️

since new Array(..).fill(undefined) is repeated couple of times it could just be utility function createArray(length)